core/src/main/java/com/linecorp/armeria/server/thrift/ThriftCallService.java

```diff
@@ -169,7 +167,7 @@ private static void invokeSynchronously(
                 return;
             }
 
-            try (PushHandle ignored = RequestContext.push(ctx)) {
+            try {
```

Comment:
Seems possible to remove one more usage from TomcatService. https://github.com/line/armeria/blob/master/tomcat/src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatService.java#L408